src/api/tests/feature_tests/steps/endpoint_lambda_mapping.py

```diff
@@ -41,6 +42,7 @@ def get_endpoint_lambda_mapping() -> ENDPOINT_LAMBDA_MAPPING:
             "Device?device_type={device_type}": api.searchDevice.index,
             "Device?device_type={device_type}&foo={foo}": api.searchDevice.index,
             "Device?foo={foo}": api.searchDevice.index,
+            "ProductTeam/{product_team_id}/Product/{product_id}": api.readCpmProduct.index,
```

Comment:
Probably need a test for this.

def test_parse_path_read_product():
    with api_lambda_environment_variables():
        import api.readCpmProduct.index

        endpoint_lambda_mapping = get_endpoint_lambda_mapping()

    assert parse_api_path(
        method="GET",
        path="ProductTeam/123/Product/456",
        endpoint_lambda_mapping=endpoint_lambda_mapping,
    ) == ({"product_team_id": "123", "product_id": "456"}, {}, api.readCpmProduct.index)